modules/ROOT/pages/sync/auth.adoc

```diff
@@ -1,13 +1,24 @@
 include::{partialsdir}/attributes.adoc[]
 
-= Binding the {sync-service} service to a Keycloak instance
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: sync_auth
 
-This section will help you to bind a Keycloak service to the {sync-service} service.
+[id='{context}_authentication']
+= Authentication
+
+The {sync-service} by default do not provide any any authentication and authorization mechanism.
+Users can configure {sync-service} to work with {keycloak-service} to provide authentication and 
+authorization for their applications.
 
 == Prerequisites
 
 * Ensure {sync-service} is xref:sync/provisioning.adoc[provisioned].
 * Ensure {keycloak-service} is xref:keycloak/provisioning.adoc[provisioned]
+* Ensure {keycloak-service} is xref:keycloak/coding.adoc[configured] to work with your mobile applications
+
+== Binding the {sync-service} service to a {keycloak-service} instance
```

Comment:
Best practice is to captialize headings, except for words like `is, a, the, to` etc.

Should be:

> Binding the {sync-service} Instance to a {keycloak-service} Instance.

Reply:
fyi, there's a debate about that  style issue, Dara describes the style called Titlecase, more likely we're going with Sentence case, which is what is there currently